R/learn.R

```diff
@@ -119,9 +121,12 @@ mssql_learn_query <- function() { # taken directly from {datamodelr}
       and ind_col.column_id = cols.column_id
     left outer join sys.systypes [types] on
       types.xusertype = cols.system_type_id
+  where schema_name(tabs.schema_id) = %s
```

Comment:
Perhaps:

```suggestion
  where tabs.schema_id = schema_id(%s)
```

In the worst case it doesn't change anything, in the best case the query is a bit faster.